### PR TITLE
[refactor] - Remove GMS GraphQL Service

### PR DESCRIPTION
**Pull request by arunvasudevan**
## Checklist
- [X ] The PR conforms to DataHub's [Contributing Guideline](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [X] Tests for the changes have been added/updated (if applicable)
- [X] Docs related to the changes have been added/updated (if applicable)

Back in Jan'21 when GMS did not have a graphql endpoint this service was added, now that GMS itself hosts both Rest.li and GraphQL endpoints there i snot reason for this service to exist. So, cleaning it up.

Also, found a JQ dependency while installing datahub so added to the quickstart.

### CODE REVIEW

**Comment by arunvasudevan** [comment]
Looks like flaky tests..need a re-run

**Comment by favealo** [comment]
Help me

**Comment by jjoyce0510** [comment]
Yayyyyy! Thank you Arun!

File: docs/quickstart.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 To deploy a new instance of DataHub, perform the following steps.
 
-1. Install [docker](https://docs.docker.com/install/) and [docker-compose](https://docs.docker.com/compose/install/) (if
+1. Install [docker](https://docs.docker.com/install/), [jq](https://stedolan.github.io/jq/download/) and [docker-compose](https://docs.docker.com/compose/install/) (if
```
Comment (jjoyce0510) [inline docs/quickstart.md:7]: Nice!!

**Review by jjoyce0510 — APPROVED** [review]
Overall this LGTM. Thank you so much @arunvasudevan.

**Comment by jjoyce0510** [comment]
Seems that CI is failing - once that is resolved this should be good

**Comment by arunvasudevan** [comment]
Sure!

**Comment by arunvasudevan** [comment]
@jjoyce0510 CI Looks good now.

**Comment by jjoyce0510** [comment]
LGTM. Thanks again!

**Review by shirshanka — APPROVED** [review]

**Pull request merged**